main.rkt

```diff
@@ -22,6 +25,9 @@
 
 ;; =============================================================================
 
+(define (square n)
+  (expt n 2))
+
```

Comment:
Wow.. I searched and couldn't din it. Why the hell is it under "number theory"? But yea if its already there let's use that. I'd also like to export unicode synonyms for most of the constants and special functions like gamma, psi, von-mangoldt, moebius, etc